src/interfaces/models.js

```diff
@@ -69,7 +69,7 @@ export const BlobMetadataModel = new Schema({
 	processingInfo: {
 		transformationError: {},
 		numberOfRecords: {type: Number, required: true, default: 0},
-		failedRecords: [new Schema({}, {_id: false})],
+		failedRecords: [],
```

Comment:
The objects must not have the `_id` property which is automatically created by Mongoose. Perhaps using a "mixed" schema will work: https://mongoosejs.com/docs/schematypes.html#mixed

Reply:
By removing `new Schema()` constructor and just by leaving `[ ]` empty array Mongoose does not generate `_id` to `failedRecords` array objects. I checked blob read on no error case and error case and also dummy-importter logs to make sure no extra id's are generated.